executor/executor.go

```diff
@@ -1607,7 +1655,8 @@ func (e *SortExec) Next() (*Row, error) {
 	return row, nil
 }
 
-// TopnExec implements a top n algo.
+// TopnExec implements a top n algorithm, it is built from a select statement with ORDER BY and LIMIT.
```

Comment:
TopnExec implements a Top-N algorithm and it is built from a select statement with ORDER BY and LIMIT.